lib/helpers.rb

```diff
@@ -65,15 +65,15 @@ def un_flag_as_abuse(obj)
     raise ArgumentError, t(:user_id_is_required) unless user
     if params["all"]
       # If historical abuse flags didn't exist so far, increment them.
-      obj.author.update_stats_for_course(obj.course_id, inactive_flags: 1) unless obj.historical_abuse_flaggers
+      obj.author.update_stats_for_course(obj.course_id, inactive_flags: 1) unless obj.historical_abuse_flaggers.empty?
```

Comment:
This should be `if obj.abuse_flaggers.empty?` instead of `unless`, if I understand correctly.